src/main/java/frc/robot/commands/TeleopCmd.java

```diff
@@ -42,14 +52,14 @@ public void execute() {
       // Follow standard order to use arcade driving preset
       // Get the axises and apply speed limits
       double turn = slrForTurn.calculate(turns * Constants.SPEEDLIMIT_TURN);
-      double speed = slrForDrive.calculate(speeds * Constants.SPEEDLIMIT_SPEED);
+      double speed = slrForDrive.calculate(speeds * Constants.SPEEDLIMIT_SPEED*getGear());
```

Comment:
So this ends up being speeds * 0.95 * 1 for high gear/speed and speeds * 0.95 * 0.55 for low gear/speed.  

Is `SPEEDLIMIT_SPEED` needed if instead getGear() returns 0.95 for high speed and 0.52 (0.95 * 0.55) for low speed?